Server/client-config.sh

```diff
@@ -116,8 +116,8 @@ if [ "$reKey" == "" ]; then
 	echo command=\"/var/bluesky/.ssh/wrapper.sh\",no-port-forwarding,no-X11-forwarding,no-agent-forwarding,no-pty `cat /usr/local/bin/BlueSkyConnect/Server/blueskyd.pub` > /usr/local/bin/BlueSkyConnect/Client/.ssh/authorized_keys
 
 	# create server.plist
-	hostKey=`ssh-keyscan -t ed25519 -p 3122 localhost | awk '{ print $2,$3 }'`
-	hostKeyRSA=`ssh-keyscan -t rsa -p 3122 localhost | awk '{ print $2,$3 }'`
+	hostKey=`ssh-keyscan -t ed25519 -p 3122 "$hostName" | awk '{ print $2,$3 }'`
```

Comment:
had an issue after updating to Ubuntu 20 within docker that ssh-keyscan on localhost/127.0.0.1 dod not work. changed to use actual hostname which works well - just making note as this change is not docker specific and i have not tested on a real server.

Reply:
fixed root cause of issue - this has been reverted